lib/shoulda/matchers/action_controller/permit_matcher.rb

```diff
@@ -62,9 +62,10 @@ module ActionController
       #             password: 'password'
       #           }
       #         }
-      #         should permit(:first_name, :last_name, :email, :password).
+      #         assert_accepts permit(:first_name, :last_name, :email, :password).
       #           for(:create, params: params).
-      #           on(:user)
+      #           on(:user),
+      #           subject
```

Comment:
I don't use Minitest, but now that I think about it, I guess it's conventional to leave out the parens for `assert_*` methods.